debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnection.java

```diff
@@ -67,13 +67,13 @@ public class PostgresConnection extends JdbcConnection {
 
     private static final Pattern FUNCTION_DEFAULT_PATTERN = Pattern.compile("^[(]?[A-Za-z0-9_.]+\\((?:.+(?:, ?.+)*)?\\)");
     private static final Pattern EXPRESSION_DEFAULT_PATTERN = Pattern.compile("\\(+(?:.+(?:[+ - * / < > = ~ ! @ # % ^ & | ` ?] ?.+)+)+\\)");
+    public static final String DEFAULT_PROTOCOL = "postgresql";
+    public static final String DEFAULT_DRIVER_CLASS_NAME = org.postgresql.Driver.class.getName();
     private static Logger LOGGER = LoggerFactory.getLogger(PostgresConnection.class);
 
-    private static final String URL_PATTERN = "jdbc:postgresql://${" + JdbcConfiguration.HOSTNAME + "}:${"
+    private static final String URL_PATTERN_WITHOUT_PROTOCOL = "${" + JdbcConfiguration.HOSTNAME + "}:${"
```

Comment:
Is not it possible to pass the protocol as pattern placeholder?

Reply:
I need to change the JDBCConfiguration to do this, which is widely used, so it's simpler to me to do like this :)